.github/workflows/release-ghcr.yaml

```
@@ -12,7 +12,34 @@ env:
   REGISTRY_AUTH_FILE: ./skopeo-registry-auth-file.json
 
 jobs:
+  wait-for-hydra:
+    name: "Wait for hydra check-runs"
+    runs-on: ubuntu-latest
+    steps:
+    - name: Waiting for ci/hydra-build:required to complete
+      run: |
+        while [[ true ]]; do
+          # Get the list of check runs for the latest commit
+          check_runs_url="https://api.github.com/repos/$GITHUB_REPOSITORY/commits/$GITHUB_SHA/check-runs"
+          check_runs=$(curl --silent --header "Authorization: token ${{ secrets.GITHUB_TOKEN }}" --header "Accept: application/vnd.github.v3+json" $check_runs_url)
+  
+          # Filter to get the specific check run you're interested in and its conclusion
+          conclusion=$(echo "$check_runs" | jq -r '.check_runs[] | select(.name == "ci/hydra-build:required") | .conclusion')
```

```suggestion
          conclusion=$(gh api repos/$GITHUB_REPOSITORY/commits/$$GITHUB_SHA/check-runs --jq '.check_runs[] | select(.name == "ci/hydra-build:required") | .conclusion')
```
should work the same. Do we get to see the pending checks?